### PR TITLE
Improve AEC by always activated. 

### PR DESCRIPTION
**Pull request by Juniverse**

### CODE REVIEW

File: Runtime/Scripts/Audio/AudioCapture.cs [hunk 1]
```diff
@@ -421,15 +434,14 @@ protected virtual IEnumerator Collect()
                     targetName = charName
                 });
             }
-            yield return new WaitForSecondsRealtime(0.1f);
         }
         protected virtual IEnumerator OutputData()
```
Comment (mattkwilson) [inline Runtime/Scripts/Audio/AudioCapture.cs:438]: This no longer needs to be an IEnumerator.

File: Runtime/Scripts/Audio/AudioCapture.cs [hunk 2]
```diff
@@ -32,7 +32,7 @@ public class AudioCapture : MonoBehaviour
         [SerializeField] protected MicSampleMode m_SamplingMode = MicSampleMode.NO_FILTER;
         [Tooltip("Hold the key to sample, release the key to send audio")]
         [SerializeField] protected KeyCode m_PushToTalkKey = KeyCode.None;
-        [Range(5, 30)][SerializeField] protected float m_PlayerVolumeThreshold = 10f;
+        [Range(0, 30)][SerializeField] protected float m_PlayerVolumeThreshold = 10f;
```
Comment (mattkwilson) [inline Runtime/Scripts/Audio/AudioCapture.cs:35]: We should lower the default value (set to 6).

**Review by mattkwilson — APPROVED** [review]

File: Runtime/Scripts/Audio/AudioCapture.cs (hunk 1 above)
Comment (Juniverse) [reply Runtime/Scripts/Audio/AudioCapture.cs:438]: I'll update in the next version

File: Runtime/Scripts/Audio/AudioCapture.cs (hunk 2 above)
Comment (Juniverse) [reply Runtime/Scripts/Audio/AudioCapture.cs:35]: Let's update in the next version

**Pull request merged**